src/components/buttons/button-table-edit.jsx

```diff
@@ -96,11 +96,12 @@ class ButtonTableEdit extends React.Component {
 	_createTable = () => {
 		const editor = this.context.editor.get('nativeEditor');
 		const tableUtils = new CKEDITOR.Table(editor);
+		const {cols, rows} = this.state;
 
 		tableUtils.create({
 			attrs: this.props.tableAttributes,
-			cols: this.state.cols,
-			rows: this.state.rows,
+			cols: cols < MINIMUM_GRID_VALUE ? MINIMUM_GRID_VALUE : cols,
+			rows: rows < MINIMUM_GRID_VALUE ? MINIMUM_GRID_VALUE : rows,
```

Comment:
Seeing as you used a `Math` function for this before (ie. on the deleted line 125) you should probably use it here too.

But note, the old version was wrong (probably because you used the _untested snippet_ that I suggested [here](https://github.com/liferay/alloy-editor/pull/1409#discussion_r459895177)). Please remember that suggestions like that are often just typed into a text field in a browser on GitHub and almost certainly haven't been tested beforehand; you need to critically evaluate each suggestion and validate that it actual does what it is supposed. Should have been `Math.max`, not `Math.min`.

```js
cols: Math.max(MINIMUM_GRID_VALUE, cols),
rows: Math.max(MINIMUM_GRID_VALUE, rows),
```

Reply:
☝️ that one is also an untested snippet.

Reply:
I tested this snippet and looks fine ;) Updated!